src/components/shared/Col.js

```diff
@@ -0,0 +1,34 @@
+import { useResponsive } from 'hooks/useResponsive'
+import { Box } from 'grommet'
+import styled, { css } from 'styled-components'
+
+const Wrapper = styled(Box)`
+  ${({ flexValue }) => css`
+    flex: ${flexValue};
+  `}
+
+  ${({ viewport }) =>
+    viewport === 'small' &&
+    css`
+      flex: 1 1 auto;
+      + div {
```

Comment:
We shouldn't be applying styles to a sibling div here. There are other ways to accomplish what this does that allows for more transparency in how / when styles are applied.

Reply:
Previously discussed and the wrapper for this component was removed for PR https://github.com/AlexsLemonade/refinebio-web/pull/32 and up (noted [above](https://github.com/AlexsLemonade/refinebio-web/pull/31#issuecomment-1329595464)), thank you!